randovania/games/am2r/json_data/header.txt

```diff
@@ -218,7 +218,8 @@ Templates
       All of the following:
           Any of the following:
               # Item requirements
-              Missiles ≥ 30 or Super Missiles ≥ 6
+              Missiles ≥ 30
+              Super Missiles ≥ 6 and Combat (Intermediate)
```

Comment:
Shouldn't the combat be on the missiles?

Reply:
Yes, it should be.  Fixed.